### ISSUE TITLE
Suggestion: Rename Kv to Chamelon_kv

### ISSUE THREAD

**Issue by reynir** [yomimono/chamelon#14]
The mirage module is named `Kv`. Perhaps it can be renamed `Chamelon_kv` so as to make it clear it's a Chamelon KV device. Obviously a breaking change :/

**Comment by yomimono** [comment]
Possibly it should be wrapped so as to be `Chamelon.Kv`? That does feel a bit more natural to me. I agree that `Kv` seems a bit presumptuous.

**Comment by reynir** [comment]
Yes, I think that works, too. One "concern" could be if you would want to release the mirage code separately at some point (but then you could as well change the name again).
